lib/db.js

```diff
@@ -476,43 +467,43 @@ DB.prototype._createTable = function (keyspace, schema, tableName) {
         + dbu.cassID(keyspace + '_' + tableName) + ' (';
     for (var attr in schema.attributes) {
         var type = schema.attributes[attr];
-        cql += dbu.cassID(attr) + ' ';
+        sql += dbu.cassID(attr) + ' ';
         switch (type) {
-        case 'blob': cql += 'blob'; break;
-        case 'set<blob>': cql += 'set<blob>'; break;
-        case 'decimal': cql += 'decimal'; break;
-        case 'set<decimal>': cql += 'set<decimal>'; break;
-        case 'double': cql += 'double'; break;
-        case 'set<double>': cql += 'set<double>'; break;
-        case 'float': cql += 'float'; break;
-        case 'set<float>': cql += 'set<float>'; break;
-        case 'boolean': cql += 'boolean'; break;
-        case 'set<boolean>': cql += 'set<boolean>'; break;
-        case 'int': cql += 'int'; break;
-        case 'set<int>': cql += 'set<int>'; break;
-        case 'varint': cql += 'varint'; break;
-        case 'set<varint>': cql += 'set<varint>'; break;
-        case 'string': cql += 'text'; break;
-        case 'set<string>': cql += 'set<text>'; break;
-        case 'timeuuid': cql += 'timeuuid'; break;
-        case 'set<timeuuid>': cql += 'set<timeuuid>'; break;
-        case 'uuid': cql += 'uuid'; break;
-        case 'set<uuid>': cql += 'set<uuid>'; break;
-        case 'timestamp': cql += 'timestamp'; break;
-        case 'set<timestamp>': cql += 'set<timestamp>'; break;
-        case 'json': cql += 'text'; break;
-        case 'set<json>': cql += 'set<text>'; break;
+        case 'blob': sql += 'blob'; break;
+        case 'set<blob>': sql += 'set<blob>'; break;
+        case 'decimal': sql += 'decimal'; break;
+        case 'set<decimal>': sql += 'set<decimal>'; break;
+        case 'double': sql += 'double'; break;
+        case 'set<double>': sql += 'set<double>'; break;
+        case 'float': sql += 'float'; break;
+        case 'set<float>': sql += 'set<float>'; break;
+        case 'boolean': sql += 'boolean'; break;
+        case 'set<boolean>': sql += 'set<boolean>'; break;
+        case 'int': sql += 'int'; break;
+        case 'set<int>': sql += 'set<int>'; break;
+        case 'varint': sql += 'varint'; break;
+        case 'set<varint>': sql += 'set<varint>'; break;
+        case 'string': sql += 'text'; break;
+        case 'set<string>': sql += 'set<text>'; break;
+        case 'timeuuid': sql += 'timeuuid'; break;
+        case 'set<timeuuid>': sql += 'set<timeuuid>'; break;
+        case 'uuid': sql += 'uuid'; break;
+        case 'set<uuid>': sql += 'set<uuid>'; break;
+        case 'timestamp': sql += 'timestamp'; break;
+        case 'set<timestamp>': sql += 'set<timestamp>'; break;
+        case 'json': sql += 'text'; break;
+        case 'set<json>': sql += 'set<text>'; break;
         default: throw new Error('Invalid type ' + type
                      + ' for attribute ' + attr);
         }
         if (statics[attr]) {
-            cql += ' static';
+            sql += ' static';
```

Comment:
Right. I didn't actually mean 'disregard statics entirely', I was merely referring to the keyword (in hindsight, rereading my comment it does sound like 'don't care about statics'). I vote for a separate table + foreign key connections from the main (data) table.